lib/push.js

```diff
@@ -218,7 +218,7 @@ module.exports = function (log, db, config) {
         }
       }))
       var options = { data: data, TTL: TTL_DEVICE_DISCONNECTED }
-      return this.pushToDevice(uid, idToDisconnect, 'deviceDisconnected', options)
```

Comment:
Oh right, of course! It's been a while since I've dug around in this file, thanks for reminding me.